includes/functions/functions_categories.php

```diff
@@ -793,15 +793,18 @@ function zen_get_category_description($category_id, $language_id = null): string
  * @param $category_id
  * @return string
  */
-function zen_get_categories_image($category_id) {
+function zen_get_categories_image($category_id): string
+{
     global $db;
 
-    $sql = "SELECT categories_image FROM " . TABLE_CATEGORIES . " WHERE categories_id= " . (int)$category_id;
-    $result = $db->Execute($sql);
+    $sql = "SELECT categories_image FROM " . TABLE_CATEGORIES . " WHERE categories_id = " . (int)$category_id;
+    $result = $db->Execute($sql, 1);
 
-    if ($result->EOF) return '';
+    if ($result->EOF) {
+        return '';
+    }
 
-    return $result->fields['categories_image'];
+    return (string)$result->fields['categories_image'];
```

Comment:
Given that the situation we're "preventing" is a `null` being returned, would it be more human-intuitive (for future looking at the code) to use null-coalesce here?

ie: 
```php
return $result->fields['categories_image'] ?? '';
```

(technically that could allow the deletion of the ->EOF check too, but I'm not opposed to keeping that)

Reply:
I'd considered that, but felt that simply casting to a string was more 'obvious' and potentially more performant.